src/easylink/step.py

```diff
@@ -851,3 +810,196 @@ def _update_slot_mappings(subgraph: dict[str, Any]) -> dict[str, list[SlotMappin
         input_mappings = subgraph["input_slot_mappings"]
         output_mappings = subgraph["output_slot_mappings"]
         return {"input": input_mappings, "output": output_mappings}
+
+
+class ConfigurationState(ABC):
```

Comment:
I simply moved this from above - I have not yet changed the docstrings (separate PR)